README.md

```diff
@@ -108,6 +108,20 @@ These are the fields that are turned into TemplateFields when the YAML files are
 *Note:*
 *All markdown formatting, HTML tags, or jinja passed in as part of the content of variables will be escaped.*
 
+Development
+-----------
+
+A local checkout of the frameworks repo can be shared with locally-running services (i.e. frontend applications)
+as follows:
+
+- from this repo, run `bower link`
+- from each app, run `bower link digitalmarketplace-frameworks`
+
+Then, whenever the framework content is changed:
+
+- on each frontend app, rebuild the static files to include the new content by running `make frontend_build` (NB this step also required before deployment to PaaS via `cf push`)
```

Comment:
It doesn't seem to work! As in, the watching bit doesn't work - it does actually copy the updated framework content on first run...